install-deps.sh

```diff
@@ -12,7 +12,7 @@
 #  License as published by the Free Software Foundation; either
 #  version 2.1 of the License, or (at your option) any later version.
 #
-set -e
+set -ex
```

Comment:
This is going to be extremely verbose for many.

For occasional debugging, [what about](https://unix.stackexchange.com/questions/536263/turn-on-xtrace-with-environment-variable):
```bash
env SHELLOPTS=xtrace ./install-deps.sh
```
Or:
```bash
[ -n "$DEBUG" ] && set -x
```

Reply:
ooops, I forgot to remove it